server/app/_components/TodoEditor.tsx

```diff
@@ -59,11 +73,41 @@ const TodoEditor = ({ editTargetTodo, onSubmit }): JSX.Element => {
           <button
             type="submit"
             className="flex w-full justify-center rounded-md bg-indigo-600 px-3 py-1.5 text-sm font-semibold leading-6 text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
-            onClick={() => onSubmit(todo)}
+            onClick={() => openDialog("確認画面","ToDoリストに追加しますか？")}
           >
             保存
           </button>
         </div>
+
+        
+        <div>
+          <dialog 
+            id="myDialog"
+            className="fixed inset-0 m-auto w-fit h-fit p-6 rounded-lg shadow-lg">
+            <h2 className="text-5xl font-bold text-black-400">
+              ダイアログのタイトル挿入
+            </h2>
+            <p className="text-3xl font-bold text-black-400">
+              ダイアログのメッセージ挿入
+            </p>
```

Comment:
宣言した要素は、JSX(return)内においてブレース（鍵括弧）で囲えばそのまま反映できます。

```suggestion
            <h2 className="text-5xl font-bold text-black-400">
              {confirmTitle}
            </h2>
            <p className="text-3xl font-bold text-black-400">
              {confirmMessage}
            </p>
```